### ISSUE TITLE
`schema.save()` does not handle part tables correctly

### ISSUE THREAD

**Issue by shenshan** [datajoint/datajoint-python#821]
![image](https://user-images.githubusercontent.com/2539832/97061982-c19eb180-154d-11eb-8ac2-39f8aa2a5fcc.png)
It should be `Segmentation.Roi`

**Comment by dimitri-yatsenko** [comment]
What does `Trace.describe()` show for that table?

**Comment by ttngu207** [comment]
I've encountered another bug in `schema.save()` - getting the name of part-table incorrectly

table = '`ephys_v2_analysis`.`__task_tracking__processed_tracking_member`'

here, the part-table's `class_name` should be `processed_tracking_member` but was returned as the name of the master `task_tracking`

Relevant code here: https://github.com/datajoint/datajoint-python/blob/143ba7b55fd292a0737e72d058670bcb5c6db064/datajoint/schemas.py#L261

In this block:
```
            if tier == 'Part':
                class_name = class_name.split('__')[1]
                indent += '    '
```

I believe should change to: `class_name = class_name.split('__')[-1]`

**Comment by ttngu207** [comment]
@dimitri-yatsenko Do you think this fix can go as part of the 0.13 pre-release. I do need this fix quite urgently

**Comment by dimitri-yatsenko** [comment]
Yep, will issue the PR today. Yes to include in 0.13.
